enterprise/web.jspparser/src/org/netbeans/modules/web/jspparser/ParserServletContext.java

```diff
@@ -660,6 +660,12 @@ public Dynamic addServlet(String string, Servlet srvlt) {
     public Dynamic addServlet(String string, Class<? extends Servlet> type) {
         return null;
     }
+    
+    @Override
+    public Dynamic addJspFile(String servletName, String jspFile) {
+        System.err.println("** addJspFile(servletName,jspFile)");
```

Comment:
Should this be a logger or even completely removed?

Reply:
It is not necessary.